packages/atomic/package.json

```diff
@@ -83,6 +83,7 @@
     "autoprefixer": "10.4.0",
     "axe-core": "4.3.5",
     "babel-loader": "8.2.3",
+    "coveo.analytics": "^2.18.53",
```

Comment:
How can we ensure this version and this version in Headless is the same? Can we reference the types from the node_modules instead?

Reply:
> How can we ensure this version and this version in Headless is the same?

Since we only need three simple types, this isn't strictly necessary.

> Can we reference the types from the node_modules instead?

If we do so, Cypress seems to prevent us from importing it from `node_modules/coveo.analytics/src`, so we have to extract the types using generic types like `Parameters<>`. I could try again though, since `src` seems to be working when installed directly and importing from this file.